tensorflow/core/kernels/gpu_prim.h

```diff
@@ -80,11 +80,29 @@ struct NumericTraits<tensorflow::bfloat16>
 }  // namespace cub
 #elif TENSORFLOW_USE_ROCM
 #include "rocm/include/hipcub/hipcub.hpp"
+#include "rocm/rocm_config.h"
 namespace gpuprim = ::hipcub;
 
 // Required for sorting Eigen::half and bfloat16.
 namespace rocprim {
 namespace detail {
+#if (TF_ROCM_VERSION >= 50200)
```

Comment:
Can we separate TF fixes out?

Reply:
yes, split it into multiple PRs. Please see the list below.